README.md

```diff
@@ -230,6 +230,7 @@
 |[PointCloud](https://point.cloud) | SENSORS | 2017 | US |4D imaging sensors |
 |[Polaris EO](https://polariseo.com) | PHOTONICS | 2021 | US |Ferrolectric based electro-optic modulators |
 |[Polyn Technology](https://polyn.ai) | AI | 2019 | IL |Neuromorphic Analog Signal Processor chips |
+|[Positron](https://positron.ai) | AI | 2023 | US |AI accelerators for Transformers |
```

Comment:
Not adding positron

Reply:
Why? In recent articles they announced plans for an ASIC.

Reply:
Andreas, I am not going to engage in public discussion regarding decisions. Inclusion is at my discretion.